src/main.py

```diff
@@ -59,6 +59,25 @@ def print_exception_as_comment(
         print("*/")
 
 
+def get_next_func(
+    names: List[str], functions: Dict[str, Function], globalinfo: GlobalInfo
+) -> Optional[Function]:
```

Comment:
Maybe this could be turned into an iterator, `yield`'ing `Function`s instead of returning one `Optional[Function]` at a time? 

Can you add a comment describing what this function is doing/why?